app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/channels/DefaultChannelAdapter.kt

```diff
@@ -65,7 +66,18 @@ class DefaultChannelAdapter(
             }
 
             itemView.setOnLongClickListener {
-                onTileLongClick?.invoke(tile)
+                val builder = AlertDialog.Builder(context)
+                builder.setTitle(tile.url)
+
+                builder.setCancelable(true)
+                builder.setPositiveButton(
+                        context.resources.getString(R.string.homescreen_tile_remove)
+                ) { dialog, _ ->
+                    _removeEvents.onNext(tile)
+                    dialog.dismiss()
+                }
+
+                builder.create().show()
```

Comment:
This is much, much nicer than our current solution.  I'll be happy to have this in the codebase.